components/brave_wallet/browser/brave_wallet_prefs.cc

```diff
@@ -208,6 +208,9 @@ void RegisterProfilePrefsForMigration(
   // Added 04/2023
   registry->RegisterBooleanPref(kBraveWalletSolanaTransactionsV0SupportMigrated,
                                 false);
+
+  // Added 06/2023
```

Comment:
probably 07/2023 by the time it is merged.

Reply:
this should be synced with https://github.com/brave/brave-core/pull/19072/files#diff-bd4f0ad9aec142c9ec088e9b4044477453f7bb370cf7dc1c050a721281d063a2R266